netmaster/k8snetwork/networkpolicy.go

```diff
@@ -391,8 +803,626 @@ func InitK8SServiceWatch(listenURL string, isLeader func() bool) error {
 		npLog.Fatalf("failed to init K8S client, %v", err)
 		return err
 	}
-	kubeNet := k8sContext{contivClient: contivClient, k8sClientSet: k8sClientSet, isLeader: isLeader}
+	//nwoPolicyDb := make(map[string]k8sNetworkPolicy, 0)
+	kubeNet := k8sContext{
+		contivClient:  contivClient,
+		k8sClientSet:  k8sClientSet,
+		isLeader:      isLeader,
+		networkPolicy: make(map[string]*k8sNetworkPolicy, 0),
+		//lookup table for Configured Network;
+		network: make(map[string]bool, 0),
+		//lookup table for Configured Policy per EPG
+		defaultPolicyPerEpg: make(map[string]string, 0),
+		epgName:             make(map[string]bool, 0),
+		policyPerEpg:        make(map[string]map[string][]string, 0),
+		//policyRules:          make(map[string][]string, 0),
+		nwPolicyPerNameSpace: make(map[string][]*k8sNetworkPolicy, 0),
+	}
+
+	//Trigger default epg : = default-group
+	kubeNet.createEpgInstance(defaultNetworkName, defaultEpgName)
 
 	go kubeNet.handleK8sEvents()
 	return nil
 }
+func getLabelSelector(key, val string) string {
+	return (key + "=" + val)
+}
+
+func (k8sNet *k8sContext) addNetworkPolicy(np *network_v1.NetworkPolicy) {
+	//check if given Policy already exist
+	if _, ok := k8sNet.networkPolicy[np.Name]; ok {
```

Comment:
I'm not sure about the top level network policy map as I thought network policy names can be reused across namespaces